jsonnet/telemeter/rules.libsonnet

```
@@ -160,6 +160,51 @@
                 sum(sum_over_time(cluster:capacity_cpu_cores:sum{label_node_role_kubernetes_io = ''}[1h:5m])) by (_id) / scalar(count_over_time(vector(1)[1h:5m]))
               |||,
             },
+            {
+              // returns 0 for any cluster reporting core capacity, used to improve performance of cluster:capacity_effective_cpu_cores
+              record: 'cluster:cpu_capacity_cores:_id',
+              expr: |||
+                group by(_id) (node_role_os_version_machine:cpu_capacity_cores:sum) * 0
```

(nit) the next rule only looks at rhcos nodes, we can filter away clusters without any rhcos nodes.
```suggestion
                group by(_id) (node_role_os_version_machine:cpu_capacity_cores:sum{label_node_openshift_io_os_id="rhcos"}) * 0
```